posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties.
+
+For 22.0.0.1-beta, there is a single beta package for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: this package contains all Open Liberty beta features and GA features and functions.
+
+This means that you can try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you give the beta package a try, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features and enhancements:
+
+* <<mp_5.0, MicroProfile 5.0>>
+* <<rest_client, MicroProfile Rest Client 3.0>>
+* <<jpa_persistence, Define JPA persistence properties at server scope>>
+
+
+[#mp_5.0]
+=== MicroProfile 5.0
+
+MicroProfile 5.0 enables applications to use MicroProfile APIs together with Jakarta EE 9.1. MicroProfile 5.0 does not provide any other functional updates except aligning with Jakarta EE 9.1.  MicroProfile 5.0 includes Config 3.0, Fault Tolerance 4.0, Rest Client 3.0, Health 4.0, Metrics 4.0, Open Tracing 3.0, Open API 3.0, and JWT propagation 2.0. This beta driver will be used as a compatible implementation for releasing MicroProfile 5.0.
+
+Include the following in the `server.xml` to enable the MicroProfile 5.0 feature in Open Liberty:
+
+[source, xml]
+----
+<featureManager>              
+    <feature>microProfile-5.0</feature>
+</featureManager>
+----
+
+Alternatively, you can enable the individual MicroProfile features that you need, for example:
+
+[source, xml]
+----
+<featureManager>              
+    <feature>mpOpenTracing-3.0</feature>
+</featureManager>
+----
+
+For more information about this update check out the link:https://github.com/eclipse/microprofile/releases/tag/5.0[MicroProfile 5.0 Release] on GitHub.
+
+
+[#rest_client]
+=== MicroProfile Rest Client 3.0
+
+MicroProfile Rest Client is an API intended to enable developers to write type-safe interfaces to abstract and invoke remote RESTful services. This is the 3.0 release of MicroProfile Rest Client, and it adds support for Jakarta EE 9.1 technologies - from a developer's perspective, the only change from the previous release (2.0) is the package space name change of Jakarta packages from `javax.*` to `jakarta.*`. Another change is that the Open Liberty implementation has changed from Apache CXF to RESTEasy - this change brings with it some behavior and property changes (most of which are already documented as link:{url-prefix}/docs/latest/reference/jaxrs-dif.html[differences between `jaxrs-2.1` and `restfulWS-3.0`]).  
```

Comment:
MicroProfile Rest Client is an API intended to enable developers to write type-safe interfaces to abstract and invoke remote RESTful services.

--->

MicroProfile Rest Client is an API that helps developers to write type-safe interfaces that abstract and invoke remote RESTful services.

Reply:
start a new sentence here instead of using a dash

... support for Jakarta EE 9.1 technologies. From a developer's perspective,

Reply:
check the formatting on the "javax to jakarta" references- if you want the asterisk to show up you'll need to escape it- it's not showing on the staging draft